packages/api/src/auth/verifiers/__tests__/secretKeyVerifier.test.ts

```diff
@@ -5,6 +5,14 @@ const secret = 'MY_VOICE_IS_MY_PASSPORT_VERIFY_ME'
 
 const { sign, verify } = createVerifier('secretKeyVerifier')
 
+beforeEach(() => {
+  jest.spyOn(console, 'warn').mockImplementation(jest.fn())
+})
+
+afterEach(() => {
+  jest.spyOn(console, 'warn').mockRestore()
+})
```

Comment:
I'm  a bit concerned  about doing this - isn't there a way to disable the logger during tests instead?

Reply:
The reason I'm concerned is that we might silence output that's requires a user's attention.